cli/cmd/root.go

```diff
@@ -305,6 +306,32 @@ func newProxyConfigOptions() *proxyConfigOptions {
 	}
 }
 
+func newConfig() configs {
+	globalConfig := &config.Global{
+		LinkerdNamespace: defaultNamespace,
+		CniEnabled:       false,
+		Registry:         inject.DefaultDockerRegistry,
+		Version:          version.Version,
+		IdentityContext:  nil,
+	}
+	proxyConfig := &config.Proxy{
+		ProxyImage:              &config.Image{ImageName: inject.DefaultDockerRegistry + "/proxy", PullPolicy: "IfNotPresent"},
```

Comment:
i'm not seeing docker image tags on the `linkerd-config` `proxy` obecjt. do these need to use `taggedProxyImage` and `taggedProxyInitImage`?

edge-19.2.5:

```yaml
  proxy: |
    {"proxyImage":{"imageName":"gcr.io/linkerd-io/proxy:edge-19.2.5",...
```

this branch:

```yaml
  proxy: |
    {"proxyImage":{"imageName":"gcr.io/linkerd-io/proxy",...
```

Reply:
That's right. Note however that this introduces a BC issue against the two two previous Edge releases I think. We could deal with this with some extra logic inside `taggedProxyImage()` and `taggedProxyInitImage()`. I also need to cater for the absence of `Global.Version` in `injectObjectMeta()`.
Either that, or force users to run `linkerd install` if they have any of those Edges. @siggy @klingerf what do you think?

Reply:
It's no problem from my perspective to force users to reinstall the full control plane when upgrading to the next edge release, which will overwrite the old configmap. I definitely prefer that approach to adding compatibility code at this point.